lib/veri_frame.mli

```diff
@@ -0,0 +1,15 @@
+
+open Bap.Std
+open Bap_traces.Std
+open Regular.Std
+
+type t [@@deriving bin_io, sexp]
+type set = Value.Set.t
+
+val create: name:string -> trace:set -> bil:set -> t
+
+val name: t -> string
+val trace: t -> set
```

Comment:
that basically means, that your abstraction doesn't abstract anything. You're just trying to create a pair. Also, name doesn't make big sense for me. I think, that you should revisit this module and provide answers for yourself - what it abstracts and why do you need this.